DPA Regions added to Dev Docs (#949)
helium/docs: docs/5g-on-helium/cbrs-radios.mdx

> 

Reviewer: three > four

Reviewer: this > these

Author: this whole section was pulled for now, will publish in an additional PR when explorer is updated

<div class="container" style={{ padding: 0, paddingTop: 0 }}>
  <div class="row">
    <div class="col col--3" style={{ marginBottom: 0 }}>
      <img className="homeicon" src={useBaseUrl('img/5g-on-helium/sas-success-small.png')} />
      <p>

>         The Small Cell Radio has been granted Spectrum Access by the SAS and is providing 5G

Reviewer: Please have consistent terminology for Spectrum Access.

Author: this header has been pulled to avoid the naming collisions 👍

        coverage.
      </p>
    </div>
    <div class="col col--3" style={{ marginBottom: 0 }}>
      <img className="homeicon" src={useBaseUrl('img/5g-on-helium/sas-connected-small.png')} />
      <p>
        The Small Cell Radio Spectrum Access was recently <code>None</code> and is comming back
        online, this status should be transitory.
      </p>
    </div>
    <div class="col col--3" style={{ marginBottom: 0 }}>
      <img className="homeicon" src={useBaseUrl('img/5g-on-helium/sas-none-small.png')} />

>       <p>The CBRS Spectrum access service is disabled to the Small Cell Radio. </p>

Reviewer: "to" > "for" ?

Author: "to" language is lifted directly from the SAS documentation.

      <p>
        See <a href="./cbrs-radios#spectrum-access-none-status">below</a> for more information.
      </p>
    </div>
    <div class="col col--3" style={{ marginBottom: 0 }}>
      <img className="homeicon" src={useBaseUrl('img/5g-on-helium/sas-failure-small.png')} />

>       <p>The CBRS Spectrum access service is disabled to the Small Cell Radio. </p>

Reviewer: Same as above "to" > "for" ?

Reviewer: NONE and FAILURE should not have the exact same summary description.

Author: corrected! thank you!

>       <p>The CBRS Spectrum access service is disabled to the Small Cell Radio. </p>

Author: ```suggestion
      <p>The CBRS Spectrum access Grant to the Small Cell Radio has Failed. </p>
```

:::

### Spectrum Access `FAILURE` Status

This status is alarming, but do not let it scare you.

For newly installed Small Cell Radios, this "Failure" status is likely because the registration
process has not been completed and submitted to the SAS.

> You may see information similar to this:

Author: ```suggestion
You may see an information pop up similar to this:
```

> > SAS status code: 200 SAS response message: Response data: installationParam

Reviewer: Where would they see this status code?

Author: this is only present during a SAS `FAILURE` state. top right corner of screen, but not always.
